week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -26,21 +39,36 @@ The function must:
 */
 
 function remove(arr, index) {
-  return; // complete this statement
+  let newArr = arr.slice()
```

Comment:
- clever idea to use `slice()` to copy the array.
- there is easier way: ` let newArr = [ ...arr ] `